inc/field.class.php

```diff
@@ -108,17 +108,9 @@ public static function install(Migration $migration, $version)
             $DB->buildUpdate(
                 PluginFieldsField::getTable(),
                 ['type' => 'dropdown-User'],
-                ['type' => 'dropdownusers']
+                ['type' => 'dropdownuser']
             )
         );
```

Comment:
This second migration was probably made for `dropdownoperatingsystems`.
```suggestion
        );

        $migration->addPostQuery(
            $DB->buildUpdate(
                PluginFieldsField::getTable(),
                ['type' => 'dropdown-OperatingSystem'],
                ['type' => 'dropdownoperatingsystems']
            )
        );

```

Reply:
agree with you